### PR TITLE
scaledToFit md file and image added

### PR DESCRIPTION
**Pull request by cjsdehaas**
### Description

In  \content\swiftui\concepts\viewmodifier\ I have created a folder named "terms" inside this folder I have created another folder "\scaledToFit" This folder contains a new file: scaledToFit.md. 

Aside from this, I have added an image in the media folder:
\media\scaled-to-fit.jpg

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+Title: '.scaledToFit()'
+Description: 'The scaledToFit() modifier can be used to scale the view to fit its parent while maintaining the original aspect ratio.'
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md:3]:
```suggestion
Description: 'Scales the view to fit its parent view while maintaining the original aspect ratio.'
```

File: content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/computer-science'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`scaledToFit()`** modifier can be used to scale the [view](https://www.codecademy.com/resources/docs/swiftui/views) to fit its parent while maintaining the original aspect ratio.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md:16]:
```suggestion
The **`.scaledToFit()`** modifier can be used to scale the [view](https://www.codecademy.com/resources/docs/swiftui/views) to fit its parent view while maintaining the original aspect ratio.
```

File: content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Syntax
+
+```pseudo
+struct MyStructure: View {
+    var body: some View {
+        Image("imagename")
+            The scaledToFit() modifier will be placed here.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md:24]:
```suggestion
            .scaledToFit()
```

File: content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+struct MyStructure: View {
+    var body: some View {
+        Image("imagename")
+            .resizable()
+            .scaledToFit()
+    }
+}
+```
+In the example above, the image that contains the `scaledToFit()` modifier has been scaled down to fit the width of the parent. The original aspect ratio of the image has been maintained.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md:44]:
```suggestion

In the example above, the image that contains the `scaledToFit()` modifier has been scaled down to fit the width of the parent. The original aspect ratio of the image has been maintained.
```

File: content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+![SwiftUI Modifier](https://raw.githubusercontent.com/Codecademy/docs/main/media/scaled-to-fit.jpg)
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/scaledToFit/scaledToFit.md:46]:
```suggestion
![SwiftUI ViewModifier scaledToFit](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-viewmodifier-scaledToFit.jpg)
```

Could we crop this image to only what will be displayed on the mobile screen?

**Review by yangc95 — COMMENTED** [review]
@cjsdehaas very well done! I conducted a first review. Let me know if you have any questions and once you're done looking over all the comments, please tag me.

**Comment by cjsdehaas** [comment]
@yangc95 Thank you! Yes, I have checked all of the comments. Is there anything I need to complete here now?

**Review by yangc95 — APPROVED** [review]
@cjsdehaas One last fix; could we either change the media image name or the link to match? This looks good for a second review otherwise. Thanks for those changes!

**Comment by cjsdehaas** [comment]
@yangc95 Yes, I have just changed the link to match the file name now.

**Comment by cjsdehaas** [comment]
@yangc95 Hello, I have a question. Is there something I forgot or need to do? I saw this hasn't been merged yet.

**Comment by yangc95** [comment]
@cjsdehaas thanks for your patience! I made a small change here before passing this on for a second review: https://github.com/Codecademy/docs/pull/1406/commits/78037b9a8f3257439419e01fbf710c25b566391e

After another round of reviews, I think this entry should be ready to go 🙂 TY for following up!

**Review by SSwiniarski — APPROVED** [review]
@cjsdehaas, @yangc95 Looks good to merge.

**Comment by SSwiniarski** [comment]
@cjsdehaas Congrats on the new entry! The entry can be seen live here: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/scaledToFit

It appears that you are anonymous:
image

If you'd like to become public, make sure to add your GitHub username to your Codecademy profile and set your visibility to everyone.

cc: @yangc95

**Comment by cjsdehaas** [comment]
@SSwiniarski @yangc95 Thank you both! And thanks for the heads up. I have changed the visibility settings and my Github username is filled into my profile. That should do it.

**Comment by yangc95** [comment]
@cjsdehaas It looks like it's still not visible. In which case, log back in to Codecademy using the GitHub option.

**Pull request merged**